BizHawk.Emulation.Cores/Consoles/Nintendo/Gameboy/Gambatte.IStatable.cs

```diff
@@ -88,46 +72,5 @@ private void NewSaveCoreSetBuff()
 			_savebuff = new byte[LibGambatte.gambatte_newstatelen(GambatteState)];
 			_savebuff2 = new byte[_savebuff.Length + 4 + 21 + 1];
 		}
-
-		private readonly JsonSerializer ser = new JsonSerializer { Formatting = Formatting.Indented };
-
-		// other data in the text state besides core
-		internal class TextStateData
-		{
-			public int Frame;
-			public int LagCount;
-			public bool IsLagFrame;
-			public ulong _cycleCount;
-			public uint frameOverflow;
-			public bool IsCgb;
-		}
-
-		internal TextState<TextStateData> SaveState()
-		{
-			var s = new TextState<TextStateData>();
-			s.Prepare();
-			var ff = s.GetFunctionPointersSave();
-			LibGambatte.gambatte_newstatesave_ex(GambatteState, ref ff);

```

Comment:
Same comment as other core:  Why are we removing text states from cores that actually implement them?

Reply:
ditto to wonderswan